app/src/main/java/com/orange/ods/app/ui/components/textfields/TextField.kt

```diff
@@ -148,7 +138,7 @@ fun TextFieldCodeImplementationColumn(
                     if (isSingleLine) stringRepresentation("singleLine", true)
                     if (hasTrailing) simple("trailing", "<trailing composable>")
                     if (hasCharacterCounter) {
-                        function("characterCounter", OdsComposable.OdsTextFieldCharacterCounter.name) {
+                        classInstance<OdsTextFieldCharacterCounter>("characterCounter") {
```

Comment:
`if (hasLeadingIcon) icon()` above should also be replaced with a `classInstance` of `OdsTextFieldIcon`.